openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -18,119 +18,6 @@
-                " edx_toggles.toggles.testutils.override_waffle_switch."
-            ),
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        set_custom_attribute(
-            "deprecated_waffle_utils", "WaffleSwitchNamespace.override"
-        )
-        from edx_toggles.toggles.testutils import override_waffle_switch
-
-        with override_waffle_switch(
-            LegacyWaffleSwitch(self, switch_name, module_name=__name__), active
-        ):
-            yield
-
-
-class WaffleSwitch(LegacyWaffleSwitch):
-    """
-    Deprecated class: instead, use edx_toggles.toggles.WaffleSwitch.
-    """
-
-    def __init__(self, waffle_namespace, switch_name, module_name=None):
-        super().__init__(waffle_namespace, switch_name, module_name=module_name)
-        warnings.warn(
-            "Importing WaffleSwitch from waffle_utils is deprecated. Instead, import from edx_toggles.toggles.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        set_custom_attribute(
-            "deprecated_waffle_utils", "WaffleSwitch[{}]".format(self.name)
-        )
-
-
-class WaffleFlagNamespace(LegacyWaffleFlagNamespace):
-    """
-    Deprecated class: instead, use edx_toggles.toggles.WaffleFlagNamespace.
-    """
-
-    def __init__(self, name, log_prefix=None):
-        super().__init__(name, log_prefix=log_prefix)
-        warnings.warn(
-            "Importing WaffleFlagNamespace from waffle_utils is deprecated. Instead, import from edx_toggles.toggles.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        set_custom_attribute(
-            "deprecated_waffle_utils", "WaffleFlagNamespace[{}]".format(name)
-        )
-
-
-class WaffleFlag(LegacyWaffleFlag):
-    """
-    Deprecated class: instead, use edx_toggles.toggles.WaffleFlag.
-    """
-
-    def __init__(self, waffle_namespace, flag_name, module_name=None):
-        super().__init__(waffle_namespace, flag_name, module_name=module_name)
-        warnings.warn(
-            "Importing WaffleFlag from waffle_utils is deprecated. Instead, import from edx_toggles.toggles.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        set_custom_attribute(
-            "deprecated_waffle_utils", "WaffleFlag[{}]".format(self.name)
-        )
-
-    @contextmanager
-    def override(self, active=True):
-        """
-        Deprecated method: instead, use edx_toggles.toggles.testutils.override_waffle_flag.
-        """
-        warnings.warn(
-            (
-                "WaffleFlag.override is deprecated. Instead, use"
-                " edx_toggles.toggles.testutils.override_waffle_flag."
-            ),
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        set_custom_attribute("deprecated_waffle_utils", "WaffleFlag.override")
-        from edx_toggles.toggles.testutils import override_waffle_flag
-
-        with override_waffle_flag(self, active):
-            yield
-
-
```

Comment:
Tests indicate that this change is breaking waffle_utils/viwes.py (and maybe other modules): https://build.testeng.edx.org/job/edx-platform-python-pipeline-pr/28401/testReport/junit/(root)/(empty)/Run_Tests___cms_unit___openedx_core_djangoapps_waffle_utils_tests_test_views/

Reply:
Rebased with your changes and fixed.